packages/embark-blockchain-process/src/gethClient.js

```diff
@@ -9,7 +9,7 @@ const constants = require('embark-core/constants');
 
 const DEFAULTS = {
   "BIN": "geth",
-  "VERSIONS_SUPPORTED": ">=1.8.14",
+  "VERSIONS_SUPPORTED": ">=1.9.0",
```

Comment:
Since we semi-break the version, maybe the new version should be 4.2?

Reply:
Yes, the intention is now to publish as 4.2.0, and I've updated to the PR description to that effect.